### PR TITLE
Aw fix breaking changes

### PR DESCRIPTION
**Pull request by andywong418**
Fix breaking changes to accept legacy configuration and restore memory endpoint. Also add tests to to check that this is valid.

### CODE REVIEW

File: src/controllers/info.ts [hunk 1]
```diff
@@ -15,10 +15,17 @@ export default function (server: Hapi.Server, deps: Injector) {
...
+    return {
+      freeMem: (os.totalmem() * config.maxMemoryFraction) - podManager.getMemoryUsed()
+    }
+  }
+
   async function infoHandler (request: Hapi.Request, h: Hapi.ResponseToolkit) {
     const fullMem = podManager.getMemoryUsed() * (2 ** 20) / os.totalmem() >= config.maxMemoryFraction
     const infoResp: HostInfo = {
       fullMem,
+      acceptingUploads: fullMem,
```
Comment (traviscrist) [inline src/controllers/info.ts]: Shouldn't this be true if the memory is not full?

**Comment by spearl** [comment]
Remember to squash your commits before merging!

**Comment by spearl** [comment]
This closes #71

**Comment by steffenmandrup** [comment]
Shouldnt memory go in /info ? Seems weird to have a dedicated endpoint for memory only

**Pull request merged**